clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/model/ElasticSearchEntityTagsProvider.java

```diff
@@ -342,6 +403,57 @@ private QueryBuilder applyTagsToBuilder(String namespace, Map<String, Object> ta
     }
   }
 
+  private List<EntityTags> fetchAll(String type, int scrollSize, String scrollTime) {
```

Comment:
@anotherchrisberry I believe we chatted about the elasticsearch scroll apis ... this seems to do the trick.